src/components/schema-display/schema-display.component.tsx

```diff
@@ -115,7 +116,11 @@ const SchemaDisplay: React.FC = () => {
           isOpen={isSchemaVisible}
           closeButtonAriaLabel={isSchemaVisible ? t('hide-schema-button') : t('view-schema-button')}
           onDismiss={() => setIsSchemaVisible(!isSchemaVisible)}
-          onOuterClick={() => setIsSchemaVisible(!isSchemaVisible)} >
+          onOuterClick={() => setIsSchemaVisible(!isSchemaVisible)} 
+        >
+          <Clipboard className="clipboard" data-clipboard-text={resumeSchema}>
```

Comment:
Let's use the class "clipboard" to make the button look like the Fluent UI buttons